src/api/foreground/foreground-setup-ar-protocol-links.ts

```diff
@@ -1,47 +0,0 @@
-import { sendMessage } from "@arconnect/webext-bridge";
-import { isString } from "typed-assert";
-
-// TODO: This is not enough for client-rendered pages (e.g. React apps):
-
-export async function replaceArProtocolLinks() {
```

Comment:
While this function was injected and called from `contents/ar-protocol.ts`, the `sendMessage` below will never return a result because there was no listener for `"ar_protocol`. Also, it looks like this was replaced by `protocol.handler.ts`, which uses `browser.webNavigation.onBeforeNavigate`.